src/crawlers/playwright_crawler.js

```diff
@@ -224,17 +228,17 @@ class PlaywrightCrawler extends BrowserCrawler {
     static optionsShape = {
         ...BrowserCrawler.optionsShape,
         browserPoolOptions: ow.optional.object,
-        playwrightModule: ow.optional.object,
+        launcher: ow.optional.object,
         gotoTimeoutSecs: ow.optional.number,
         navigationTimeoutSecs: ow.optional.number,
-        launchOptions: ow.optional.object,
+        launchContext: ow.optional.object,
     }
 
     constructor(options = {}) {
         ow(options, 'PlaywrightCrawlerOptions', ow.object.exactShape(PlaywrightCrawler.optionsShape));
 
         const {
-            playwrightModule,
+            launcher,
```

Comment:
I'm writing this comment after the one below, but:

I just remembered that the `launcher` should be part of `launchContext` no? Because the type includes it.